src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -4606,6 +4606,39 @@ BOOL MethodContext::repIsValidStringRef(CORINFO_MODULE_HANDLE module, unsigned m
     return value;
 }
 
+void MethodContext::recGetStringLength(CORINFO_MODULE_HANDLE module, unsigned metaTOK, int result)
+{
+    if (GetStringLength == nullptr)
+        GetStringLength = new LightWeightMap<DLD, DWORD>();
+
+    DLD key;
+    ZeroMemory(&key, sizeof(DLD)); // We use the input structs as a key and use memcmp to compare.. so we need to zero
+                                   // out padding too
+
+    key.A = (DWORDLONG)module;
```

Comment:
I copy-pasted `IsStringValid` because it has the same signature but it looks like it can be simplified a bit, e.g. we don't need `DWORDLONG` here (and in `IsStringValid`) so the container can be just `DD`